kernel/src/oops.rs

```diff
@@ -0,0 +1,84 @@
+// SPDX-License-Identifier: MPL-2.0
+
+//! Kernel oops handling.
+//!
+//! In Asterinas, the Rust panic leads to a kernel oops. The kernel oops behave
+//! as resettable exceptional control flows. If the kernel oops happens too
+//! frequently, the kernel will panic and halt the system.
```

Comment:
It is based on an ever-increasing counter. So it is "not too frequently", but "too many times".